tests/block_log.cpp

```diff
@@ -36,7 +36,7 @@ struct block_log_fixture {
       }
       else {
          eosio::chain::genesis_state gs;
-         log->reset(gs, std::make_shared<eosio::chain::signed_block>());
+         log->reset(gs, eosio::chain::signed_block::create_signed_block(eosio::chain::signed_block::create_mutable_block({})));
```

Comment:
Why this complicated?

Reply:
The idea is to make it difficult to misuse `signed_block`. We want `packed_block` data of `signed_block` to always be populated. Could create a method of `signed_block` that just does this, but since that would only be useful in tests I didn't.